torch/_functorch/aot_autograd.py

```diff
@@ -1301,7 +1301,8 @@ def flattened_joint(*args):
                     assert grad is None
             return *fw_outs, *output_gradients
 
-        fx_g = make_fx(flattened_joint)(*full_args)
+        flattened_joint._orig_mod = fx_g
```

Comment:
Can we use functools.wraps to get the orig mod? Can you also reference this note here https://github.com/pytorch/pytorch/pull/115454/files#diff-594ef3e84a8e3555bfa4a6167383949da5f321e93f602246eb9c0a797d5ecc00R617 
cc: @bdhirsh